lib/get-exporter.js

```diff
@@ -0,0 +1,116 @@
+const paginateRequest = require('./paginate-request');
+const inquirer = require('inquirer');
+const {
+  loadConfig,
+  getShallowStatus,
+  logResult,
+  logProcessing,
+  logError,
+  mapById,
+  plural
+} = require('./utils');
+const {
+  remove,
+  writeFile,
+  ensureDir
+} = require('fs-extra');
+const { merge, values, isEmpty } = require('omnibelt');
+const allSettledSerial = require('./all-settled-serial-p');
+const { rollbarLog } = require('./rollbar');
+
+const getExporter = ({
+  apiType, commandType, localStatusParams, remoteStatusParams, getData, extraQuery
+}) => {
+  const exporter = async (pattern, command = {}, loadedConfig) => {
+    await ensureDir(apiType);
```

Comment:
I would do this after you have loaded the config, cause if the config doesnt load that could mean that they need to run `losant configure` which would then generate this directory.

Reply:
And shouldnt this be commandType and not apiType?

Reply:
👍 
currently for files, experiences, and dataTables, apiType and commandType are the same thing. It probably makes more sense from a reader's perspective to use commandType.

Reply:
on line 44, you are referring to the commandType as the directory.

Reply:
haha good point. Then it makes even more sense to use it